extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -45,84 +45,84 @@ private function mockMonetizationModule() {
 	public function adContextDataProvider() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function "adContextDataProvider" has 184 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS138)